inference_cli/lib/benchmark/api_speed.py

```diff
@@ -171,7 +171,7 @@ def execute_api_request(
             batch_size=request_batch_size, duration=duration
         )
         status_code = exc.__class__.__name__
-        if issubclass(exc, requests.exceptions.HTTPError):
+        if issubclass(exc.__class__, requests.exceptions.HTTPError):
```

Comment:
you can also probably change this instaed
```suggestion
        if isinstance(exc, requests.exceptions.HTTPError):
```

Reply:
Thank you @probicheaux , this was my original approach and was asked to switch to issubclass during first code review :)